### PR TITLE
Add org.etherpad.etherpad-lite

### PR DESCRIPTION
**Pull request by SamTV12345**
### Please confirm your submission meets all the criteria

- [ ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ ] My pull request follows the instructions at [App Submission][submission].
- [ ] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

**Comment by SamTV12345** [comment]
If someone has an idea how to make Etherpad work. I am happy for any help.

In general Etherpad requires:
- current NodeJS
- npm@6 

to work. Unfortunately I don't really know how to specify that in the manifest.

File: org.etherpad.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/ether/etherpad-lite/archive/refs/tags/v1.9.4.tar.gz",
+          "dest": "etherpad-lite",
+          "sha256": "e198214acdbb57363561dee2d73f27199999af26c283723067525bd854703e12",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1,
```
Comment (bbhtt) [inline org.etherpad.json:26]: This does not belong to etherpad. Use git checker or json checker.

**Comment by bbhtt** [comment]
> In general Etherpad requires:
> 
>     * current NodeJS
> 
>     * npm@6

Use https://github.com/flathub/org.freedesktop.Sdk.Extension.node18 it has Node 18.18.1 and npm 9.8.1. If npm v6 is a hard requirement, you have to add it in the manifest.

File: org.etherpad.json (hunk 1 above)
Comment (hfiguiere) [reply org.etherpad.json:26]: well it looks like this was cut and paste from an example given the template url.

**Review by hfiguiere — COMMENTED** [review]
Flatpak and flathub are NOT suited for server software are all.

File: org.etherpad.json (hunk 1 above)
Comment (SamTV12345) [reply org.etherpad.json:26]: Yes. That is why it is a draft pull request. @Heus-Sueh asked if I could make etherpad available on FlatHub

**Comment by SamTV12345** [comment]
> Flatpak and flathub are NOT suited for server software are all.

So Flatpak is only suited for Desktop apps? I haven't used Linux in a Desktop environment which could be why I didn't hear about Flatpak before.

File: org.etherpad.json (hunk 1 above)
Comment (hfiguiere) [reply org.etherpad.json:26]: draft pull request are mean to signal "this work but still waiting on some stuff". Not mean to be for throwing thing on the wall and see if it sticks. One of the item in the checklist ask you to have built this locally.

**Comment by Heus-Sueh** [comment]
> > Flatpak and flathub are NOT suited for server software are all.
> 
> So Flatpak is only suited for Desktop apps? I haven't used Linux in a Desktop environment which could be why I didn't hear about Flatpak before.

Sorry to cause this problem, I thought Etherpad-lite was a client, but it is a server so flatpak is not recommended for this. This pull request can be closed and sorry again for the confusion.

**Pull request closed without merging**